server/src/service/OfferService.cc

```diff
@@ -8,9 +8,19 @@ OfferService::createShared()
     return std::make_shared<OfferService>();
 }
 
+void
+validateDateHTTP(UInt64 const& date_from, UInt64 const& date_to)
+{
+    OATPP_ASSERT_HTTP(date_from <= date_to,
+                      Status::CODE_400,
+                      "'date_from' cannot occur later than 'date_to'");
```

Comment:
should be changed to sth more simple like "Invalid date"

Reply:
> should be changed to sth more simple like "Invalid date"

I think, it's better to point out the error as precisely as we can.

Reply:
let's stick to simple messages. It is very different from the others stylistically

Reply:
How about "Time range cannot be negative" or "Invalid time range"